dockerfiles/pd.Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM hub.pingcap.net/bases/pd-base:v1.6.0
+FROM hub.pingcap.net/bases/pd-base:v1.6.1
 COPY pd-server /pd-server
 COPY pd-ctl /pd-ctl
+COPY pd-recover /pd-recover
```

Comment:
Please update dockerfiles in multi stage folder, thanks.

Reply:
done